report.md

```diff
@@ -0,0 +1,149 @@
+
+* **Continuous integration setup**. By making use of GitHub's actions, it was
+  possible to introduced a CI workflow within the main poliastro's repository
+  which triggers the whole validation cases every time a new piece of code is
+  merged into the main branch.
+
+
+## Obtained results and benefits
+
+Once all required poliastro's features were validated, the following results
+arose:
+
+* An error was detected in the Moon's rotational elements, which
+  which introduced numerical differences of the order of 1E-5. These proof the
+  frame validation logic to be successful.
+
+* The frame logic inside poliastro's API was proof to be right. This logic could
+  be expanded to the Astropy project, so other projects might benefit from
+  this.
+
+* Auxiliary frames, such us planetary-fixed non-parallel to ICRF, can be
+  implemented.
+
+* Groundtrack plots can be now extended to the rest of planets of the Solar
+  System.
+
+
+## Associated pull-requests 
+
+All the pull-requests generated as a result of this work are listed down below,
+being classified according to the repository in which they originated.
+
+
+**validation repository**: https://github.com/poliastro/validation
+
+* [Orekit validation first approach:](https://github.com/poliastro/validation/pull/4)
+  https://github.com/poliastro/validation/pull/4
+
+* [Create ci_actions.yml:](https://github.com/poliastro/validation/pull/7)
+  https://github.com/poliastro/validation/pull/7
+
+* [Stop tracking orekit-data.zip:](https://github.com/poliastro/validation/pull/9)
+  https://github.com/poliastro/validation/pull/9
+
+* [Enable repository dispatch from poliastro:](https://github.com/poliastro/validation/pull/11)
+  https://github.com/poliastro/validation/pull/11
+
+* [Maneuvers validation against Orekit:](https://github.com/poliastro/validation/pull/15)
+  https://github.com/poliastro/validation/pull/15
+
+* [Get rid of Orekit-based API:](https://github.com/poliastro/validation/pull/16)
+  https://github.com/poliastro/validation/pull/16
+
+* [Fix order of magnitudes:](https://github.com/poliastro/validation/pull/17)
+  https://github.com/poliastro/validation/pull/17
+
+* [Validate Hohmann 3D maneuver:](https://github.com/poliastro/validation/pull/18)
+  https://github.com/poliastro/validation/pull/18
+
+* [Bielliptic 3D maneuvers validation:](https://github.com/poliastro/validation/pull/19)
+  https://github.com/poliastro/validation/pull/19
+
+**poliastro repository**: https://github.com/poliastro/poliastro
+
+* [Fix MoonFrame W angle value:](https://github.com/poliastro/poliastro/pull/1193)
+  https://github.com/poliastro/poliastro/pull/1193
+
+* [Add GitHub actions for triggering validation repo:](https://github.com/poliastro/poliastro/pull/1051)
+  https://github.com/poliastro/poliastro/pull/1051
+
+* [Disable validation trigger in PR:](https://github.com/poliastro/poliastro/pull/1181)
+  https://github.com/poliastro/poliastro/pull/1181
```

Comment:
Perhaps we can add a small paragraph with "future work"?

Reply:
Sure!